testutils/mocks/Logger.go

```diff
@@ -0,0 +1,81 @@
+package mocks
```

Comment:
I see. I wasn't aware of `test_utils.go`. In general, we should strive for less utility-type packages/modules and be as specific as we can be. If we can group functions / types defined within our `test_utils.go` more specifically, I'd be for that. But let's not do that now. Carry on.